deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.2/kubevirt-hyperconverged-operator.v0.0.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1451 @@
+          - apiGroups:
+            - apps
+            resources:
+            - deployments
+            - daemonsets
+            - replicasets
+            - statefulsets
+            verbs:
+            - '*'
+          - apiGroups:
+            - monitoring.coreos.com
+            resources:
+            - servicemonitors
+            verbs:
+            - get
+            - create
+          - apiGroups:
+            - kubevirt.io
+            resources:
+            - '*'
+            verbs:
+            - '*'
+          - apiGroups:
+            - oauth.openshift.io
+            resources:
+            - '*'
+            verbs:
+            - '*'
+
+      deployments:
+        - name: hco-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                name: hyperconverged-cluster-operator
+            strategy: {}
+            template:
+              metadata:
+                labels:
+                  name: hyperconverged-cluster-operator
+              spec:
+                containers:
+                - command:
+                  - hyperconverged-cluster-operator
+                  env:
+                  - name: OPERATOR_IMAGE
+                    value: quay.io/kubevirt/hyperconverged-cluster-operator:latest
+                  - name: OPERATOR_NAME
+                    value: hyperconverged-cluster-operator
+                  - name: POD_NAME
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: metadata.name
+                  - name: WATCH_NAMESPACE
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: metadata.namespace
+                  image: quay.io/kubevirt/hyperconverged-cluster-operator:latest
+                  imagePullPolicy: Always
+                  name: hyperconverged-cluster-operator
+                  resources: {}
+                serviceAccountName: hyperconverged-cluster-operator
+
+
+        - name: virt-operator
+          spec:
+            replicas: 2
+            selector:
+              matchLabels:
+                kubevirt.io: virt-operator
+            strategy:
+              type: RollingUpdate
+            template:
+              metadata:
+                annotations:
+                  scheduler.alpha.kubernetes.io/critical-pod: ""
+                  scheduler.alpha.kubernetes.io/tolerations: '[{"key":"CriticalAddonsOnly","operator":"Exists"}]'
+                labels:
+                  kubevirt.io: virt-operator
+                  prometheus.kubevirt.io: ""
+                name: virt-operator
+              spec:
+                containers:
+                - command:
+                  - virt-operator
+                  - --port
+                  - "8443"
+                  - -v
+                  - "2"
+                  env:
+                  - name: OPERATOR_IMAGE
+                    value: kubevirt/virt-operator:latest
```

Comment:
Should the kubevirt image be on latest? I wonder if we should be using actual version numbers. CSV v0.0.1 also has OPERATOR_IMAGE set to kubevirt/virt-operator:latest. So an upgrade is attempted from v0.0.1 to v0.0.2, we may not see an upgrade occur for the virt-operator.

I have the same concern for other images listed using latest below.

Reply:
it shouldn't be set to latest, but I'm not sure what it should be set to. We aren't locked with kubevirt's release cycle so it's really up to us what we want to set it to for testing.  Regardless of versions, we should still be able to test the upgrade procedure through OLM.

This exposes a larger problem though that we'll need to sort out for CI...

Reply:
After testing a few days ago, I saw OLM upgrade the component operators.  What's odd is that with latest, all the pods restarted when I don't think any new containers were pulled.  It could be that OLM assumes that a new CSV means every container must be updated regardless?